detect_secrets/plugins/keyword.py

```diff
@@ -52,69 +52,85 @@
 )
 # Includes ], ', " as closing
 CLOSING = r'[]\'"]{0,2}'
+AFFIX_REGEX = r'\w*'
 DENYLIST_REGEX = r'|'.join(DENYLIST)
+# Support for suffix after keyword i.e. password_secure = "value"
+DENYLIST_REGEX = r'({denylist}){suffix}'.format(
```

Comment:
I notice that this adds "suffix support" by default to all capturing regexes; however, "prefix support" is only for `PRECEDED_BY_EQUAL_COMPARISON_SIGNS_QUOTES_REQUIRED_REGEX`. What's the reason for this?

Reply:
This is because in the other regex the secret value is always after the keyword, so the prefix is not important to us. For example: in the line `my_password = "value"` the plugin will detect `password = "value"`, even if `my_` is not included in the regex.

In the `PRECEDE_BY_EQUAL_COMPARISON_SIGNS_QUOTES_REQUIRED_REGEX` the secret value is always before the keyword, so the prefix is needed to detect the secret. For example: in the line `if ("value" == my_password) {` if the  `my_` isn't included in the line, the secret won't be detected.